packages/editor/src/components/rich-text/index.js

```diff
@@ -1059,6 +1059,17 @@ RichTextContainer.Content = ( { value, format, tagName: Tag, ...props } ) => {
 	return content;
 };
 
+/**
+ * Check if the given `RichText` value is empty on not.
+ *
+ * @param {Array} value `RichText` value.
+ *
+ * @return {boolean} True if empty, false if not.
+ */
+RichTextContainer.isEmpty = ( value ) => {
```

Comment:
Should we want to eliminate or consolidate the `isEmpty` implementation in the base `RichText` to use this one?

Reply:
Yes, that sounds good!